src/Swiping/SwipeProfiles.js

```diff
@@ -136,6 +137,50 @@ class SwipeProfiles extends React.Component {
       });
   }
 
+  async componentDidMount(){
+    const storedLocation = getCookie("location");
+    const storedLongitude = storedLocation[0];
+    const storedLatitude = storedLocation[1];
+
+    var currentLocation = await getLocation();
+    const currentLongitude = currentLocation.coords.longitude;
```

Comment:
you should check if current location has coords first. It's possible there will be an error and it won't return location so need to check for that